src/main/java/no/sikt/nva/model/content/ContentFile.java

```diff
@@ -0,0 +1,84 @@
+package no.sikt.nva.model.content;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
+import no.sikt.nva.model.content.ResourceContent.BundleType;
+
+public class ContentFile {
+
+    private String filename;
+    private BundleType bundleType;
+    private String description;
+    private String unknownType;
+
```

Comment:
Har du mulighet til å slenge på en uuid? vi kommer til å trenge det for lagring på S3, så kan like gjerne generere den med en gang. UUID::randomUUID, se slack med Rurik og meg i.

Reply:
Vet det er utenfor scopet på tasken din, men siden Rurik sa det var en oneliner så tenker jeg at det er greit å få med denne tasken i stedet for å lage knøttetask på det.

Reply:
ordnet